Dockerfile

```
@@ -1,25 +1,18 @@
 FROM docker.io/library/python:3.11
 
-# Copy firewatch package into /firewatch folder
-RUN mkdir /firewatch
-COPY . /firewatch/
+COPY pyproject.toml poetry.lock README.md /firewatch/
+COPY cli /firewatch/cli/
 
-# Install firewatch
-RUN pip install --upgrade pip
-RUN pip install /firewatch tox
+WORKDIR /firewatch
 
-# Copy the Jira config template to /templates
-RUN mkdir /templates
-COPY cli/templates/* /templates/
+ENV POETRY_HOME=/firewatch
+ENV PATH="/firewatch/bin:$PATH"
 
-# Add permissions
-RUN chgrp -R 0 /tmp && \
-    chmod -R g=u /tmp
+RUN python3 -m pip install pip poetry --upgrade \
+    && poetry config cache-dir /openshift-cli-installer \
+    && poetry config virtualenvs.in-project true \
+    && poetry config installer.max-workers 10 \
+    && poetry install
```

```suggestion
    && poetry install \
    && printf '#!/bin/bash \n poetry run firewatch $@' > /usr/bin/firewatch \
    && chmod +x /usr/bin/firewatch
```

This will allow us to use just `firewatch` rather than `poetry run firewatch`

---

ack